support/guice/src/main/java/org/apache/shiro/guice/ShiroModule.java

```diff
@@ -52,7 +52,7 @@
 
 	private Set<Destroyable> destroyables = Collections.newSetFromMap(new WeakHashMap<Destroyable, Boolean>());
     public void configure() {
-        this.binder().requireExplicitBindings();
+//        this.binder().requireExplicitBindings();

```

Comment:
@jbunting I'm not sure why this was needed, but seems to be causing problems with guice4 in the ShiroModuleTests (and the tests pass without this with both guice 3 and 4)